Core/HTTPSTransientBloomFilterSpecification.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  HTTPSTransientBloomFilterSpecification.swift
+//  Core
+//
+//  Copyright © 2018 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+public struct HTTPSTransientBloomFilterSpecification {
+    let totalEntries: Int
```

Comment:
This could be `Decodable` which would make the JSON parsing a little easier.  Can use optionals if you're worried about data missing in future versions of the file.  Data in the JSON that isn't handled is ignored by default.